### PR TITLE
MalyanLCD: Pause, resume, more ExtUI

### PR DESCRIPTION
**Pull request by xC0000005**
### Description

Implements pause and resume functionality, bringing malyan lcd functionality almost into parity with the closed source firmware. Uses ExtUI for most operations.

### Benefits

Pause, pause screen, resume work better now. 

### Related Issues

This does NOT fix the M300/V3 safety timeout issue - that will require changes to ExtUI for an OnHeating event.

### CODE REVIEW

**Comment by thinkyhead** [comment]
Question on build instructions at #11104…
> I want to add a README.md for the M200 giving instructions on the build. Is Arduino IDE still the best option? If so, then I can just itemize the steps for Arduino. But if you have steps that work with the current PlatformIO platforms, I can add those instructions too. And, if the steps are not too complicated, then we may be able to add them to the Python build script to automate the process.

**Comment by xC0000005** [comment]
I'm going to set up a  clean windows machine and see how compiling with PlatformIO goes. I switched to Arduino back when PIO's support for ST's core was horrible, but it's time to revisit that. I'll test your changes as well (the delegate solution is very clean - never thought of doing that, but it's obvious now).

**Comment by thinkyhead** [comment]
> I switched to Arduino back when PIO's support for ST's core was horrible

Arduino is a fine solution so it's good that we keep supporting it. I just want to include the details about where to get the platform / board support, how to install it in Arduino, and how to use it to do a build afterward. This is as much for our reference as users' — especially as the number of boards we're supporting is multiplying so quickly.

**Comment by xC0000005** [comment]
I tested this - command wasn't captured to the lambda according to the compiler, so I made a minor change. The LCD is, to the best of my knowledge, completely functional now.

**Comment by thinkyhead** [comment]
> I tested this - command wasn't captured to the lambda according to the compiler, so I made a minor change. The LCD is, to the best of my knowledge, completely functional now.

I'll give the `[]` capture block a quick try and see if that works.

File: Marlin/src/lcd/extui_malyan_lcd.cpp [hunk 1]
```diff
@@ -180,32 +183,18 @@ void process_lcd_eb_command(const char* command) {
...
-      char cmd[20], pos[6];
-      sprintf_P(cmd, PSTR("G1 %c%s"), axis, dtostrf(atof(command + 1) / 10.0, -5, 3, pos));
-      queue.enqueue_one_now(cmd);
-    } break;
-    default:
-      SERIAL_ECHOLNPAIR("UNKNOWN J COMMAND", command);
-      return;
+  auto move_axis = [](const auto axis) {
+    const float dist = atof(command + 1) / 10.0;
+    ExtUI::setAxisPosition_mm(ExtUI::getAxisPosition_mm(axis) + dist, axis);
```
Comment (xC0000005) [inline Marlin/src/lcd/extui_malyan_lcd.cpp:188]: In the most recent code, I'm back to getting this error:
`ork/exec /Users/jcnelson/Library/Arduino15/packages/STM32/tools/arm-none-eabi-gcc/8.2.1-1.7/bin/arm-none-eabi-g++: resource temporarily unavailable
/var/folders/2w/45wss0310nbfs7dbvg2mv01c0000gn/T/arduino_build_338636/sketch/src/lcd/extui_malyan_lcd.cpp: In lambda function:
/var/folders/2w/45wss0310nbfs7dbvg2mv01c0000gn/T/arduino_build_338636/sketch/src/lcd/extui_malyan_lcd.cpp:187:29: error: 'command' is not captured
     const float dist = atof(command + 1) / 10.0;`

**Pull request merged**